examples/create_letters_from_csv/letter.py

```diff
@@ -0,0 +1,149 @@
+        open(errors_filename, 'w') as errors:
+
+    # Print mode to screen
+    mode = lob.api_key.split('_')[0]
+    print('Sending letters in %s mode.' % mode.upper())
+
+    input_csv = csv.DictReader(input)
+    errors_csv_fields += input_csv.fieldnames
+
+    success_csv = csv.DictWriter(success, fieldnames=success_csv_fields)
+    errors_csv = csv.DictWriter(errors, fieldnames=errors_csv_fields)
+
+    # Create from_address
+    try:
+        from_address = lob.Address.create(
+            name='Your Name/Company',
+            address_line1='123 Test Avenue',
+            address_city='San Francisco',
+            address_state='CA',
+            address_zip='94107',
+        )
+    except Exception, e:
+        print('Error: %s' % str(e))
+        print('Failed to create from_address. Aborting all sends.')
+        sys.exit(1)
+
+    # Loop through input CSV rows
+    for idx, row in enumerate(input_csv):
+        if IS_HEADER and idx == 0:
+            continue
+
+        # Create letter from row
+        try:
+            letter = lob.Letter.create(
+                    description='Bill for ' + row['name'],
+                    metadata={
+                        'campaign': 'billing_statements',
+                        'csv': input_filename
+                        },
+                    to_address={
+                        'name': row['name'],
+                        'address_line1': row['address1'],
+                        'address_line2': row['address2'],
+                        'address_city': row['city'],
+                        'address_zip': row['postcode'],
+                        'address_state': row['state']
+                        },
+                    from_address=from_address.id,
+                    file=letter_html,
+                    data={
+                        'date': datetime.datetime.now().strftime("%m/%d/%Y"),
+                        'name': row['name'],
+                        'amountDue': row['amount']
+                        },
+                    color=True
+                    )
+        except Exception, e:
+            error_row = {'error': e}
+            error_row.update(row)
+            errors_csv.writerow(error_row)
+            sys.stdout.write('E')
+            sys.stdout.flush()
+        else:
+            success_csv.writerow({
+                'name':          letter.to_address.name,
+                'id':            letter.id,
+                'url':           letter.url,
+                'address_line1': letter.to_address.address_line1,
+                'address_line2': letter.to_address.address_line2,
+                'address_city':  letter.to_address.address_city,
+                'address_state': letter.to_address.address_state,
+                'address_zip':   letter.to_address.address_zip
+            })
+
+            # Print update
+            sys.stdout.write('.')
+            sys.stdout.flush()
+
+        # New lines for larger csv's
+        if idx % 10 is 9:
```

Comment:
Since a `.` is printed for each letter/line in the csv, I figured it'd be nicer to have them in rows of 10 instead of just one line since some of the csv's may be massive.